config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -25,24 +25,33 @@ latest_stable_k8s_version="1.15.0"
 latest_stable_k8s_minor_version="1.15"
 
 # We need this image because it has Docker in Docker and go.
-dind_image="gcr.io/k8s-testimages/kubekins-e2e:v20190906-fc85258-master"
+dind_image="gcr.io/k8s-testimages/kubekins-e2e:v20190917-d326d2a-master"
 
 # All kubernetes-csi repos which are part of the hostpath driver example.
 # For these repos we generate the full test matrix.
 hostpath_example_repos="
 csi-driver-host-path
 external-attacher
 external-provisioner
+external-resizer
+external-snapshotter
 node-driver-registrar
 "
 
-# kubernetes-csi repos which only need to be tested against a single
-# Kubernetes version. We generate unit, stable and alpha jobs for these,
-# without specifying a Kubernetes version.
+# kubernetes-csi repos which only need to be tested against at most a
+# single Kubernetes version. We generate unit, stable and alpha jobs
+# for these, without specifying a Kubernetes version. What the repo
+# then tests in those jobs is entirely up to the repo.
+#
+# livenessprobe is listed here because the API that it provides
+# towards Kubernetes hasn't changed much. But even that isn't actually
+# getting tested at the moment because the sidecar isn't part of
+# the csi-driver-host-path deployment and thus doesn't get tested
```

Comment:
It is? https://github.com/kubernetes-csi/csi-driver-host-path/blob/master/deploy/kubernetes-1.16/hostpath/csi-hostpath-plugin.yaml#L106

I think it's fine to move liveness probe up too